frontend/src/worker/filter.worker.ts

```diff
@@ -0,0 +1,39 @@
+import { Umbra } from '@umbra/umbra-js';
+import { getAddress } from 'src/utils/ethers';
+
+const ctx: Worker = self as any;
+
+self.addEventListener(
```

Comment:
Could you add some comments to this file and `worker.ts` with some info on their relationship, how things work, and some links to any relevant docs? I'm not really familiar with web workers, so comments would be helpful partly for my own understanding and partly to help future updates 🙂